src/client/src/containers/RightSidebar/index.tsx

```diff
@@ -147,7 +147,10 @@ class RightSidebar extends React.Component<Props, IRightSidebarState> {
         {pathname !== ROUTES.PAGE_DETAILS && (
           <div
             role="complementary"
-            className={classNames(styles.container, styles.rightViewCropped)}
+            className={classNames(styles.container, styles.rightViewCropped, {
```

Comment:
I'm a little bit confused here: we're conditionally adding rightViewCroppedSummaryPage class here in addition to rightViewCropped.
Now, rightViewCroppedSummaryPage has only `width: 400px`, but rightViewCropped has also `width: 400px`.
So, what value does it bring?
Maybe I missed something ;)

Reply:
oh! It' .rightViewCropperSummaryPage is useful for when we want to resize in the future - for ex. once the top nav bar is up, we can change the value of width to 600px or a bigger width

Reply:
Ok, thanks :)